### PR TITLE
Study definition

### PR DESCRIPTION
**Pull request by josephignace**
Common variables updated following the approved codes by the clinician. Some of these will be updated later. Prescription variables will be added later as well.

### CODE REVIEW

File: analysis/common_variables.py [hunk 1]
```diff
@@ -173,347 +173,552 @@ def generate_common_variables(index_date_variable,end_date_variable):
...
-    # ),
-    #         # ONS
-    # tmp_out_date_rem_sleep_syndrome_death=patients.with_these_codes_on_death_certificate(
-    #     rem_sleep_syndrome_icd10,
+    # Hypercholesterolaemia 
+    # Primary
+    tmp_cov_bin_hypercholesterolaemia_snomed=patients.with_these_clinical_events(
+        hypercholesterolaemia_snomed,
+        returning="date",
+        on_or_after=f"{index_date_variable}",
```
Comment (marwaalarab) [inline analysis/common_variables.py:683]: for covariates we use on_or_before don't we ? or is it on purpose for these particular ones ?

Comment (marwaalarab) [reply analysis/common_variables.py:683]: and we use - 1 day. 
for cholesterol I've been told to use between=[f"{index_date_variable}- 5years", f"{index_date_variable} -1 day"]
I think you need to check which one to use

File: analysis/common_variables.py [hunk 2]
```diff
@@ -173,347 +173,552 @@ def generate_common_variables(index_date_variable,end_date_variable):
...
+            "date": {"earliest": study_dates["pandemic_start"], "latest" : "today"},
+            "rate": "uniform",
+            "incidence": 0.1,
+        },
+    ),
+    # HES
+    tmp_cov_bin_hypercholesterolaemia_hes=patients.admitted_to_hospital(
+        returning="date_admitted",
+        with_these_diagnoses=hypercholesterolaemia_icd10,
+        on_or_after=f"{index_date_variable}",
```
Comment (marwaalarab) [inline analysis/common_variables.py:696]: same as above, on_or_before or between

**Review by marwaalarab — CHANGES_REQUESTED** [review]
Hi Jose, please see the comments. And make the changes if needed.

File: analysis/common_variables.py (hunk 1 above)
Comment (josephignace) [reply analysis/common_variables.py]: Thanks, I changed. I am using 1680 days (5 years) for precission.

**Review by marwaalarab — APPROVED** [review]

**Pull request merged**